explainers/update-rendering.md

```diff
@@ -1,13 +1,19 @@
-## `updateRendering()`
+## `renderPriority` attribute.
+
+_Note that issues referenced in this document can refer to either the
+`renderPriority` attribute or `updateRendering()` JavaScript API. The issues
```

Comment:
is updateRendering mentioned anywhere in this doc now? If not, remove?

Reply:
It's mentioned in the issues which I reference, so I just wanna keep this here as a clarification that we're talking about one and the same

Reply:
ok